package.json

```diff
@@ -17,31 +17,50 @@
     "countries",
     "ar",
     "az",
+    "be",
+    "bg",
+    "bs",
     "cs",
     "da",
     "de",
+    "el",
```

Comment:
seems to be a duplicate?

Reply:
Yeah, I thought I lost it after sorting alphabetically. I will remove the one below. It's out of order.